src/components/ha-sortable.ts

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable lit/prefer-static-styles */
+import { html, LitElement, nothing, PropertyValues } from "lit";
+import { customElement, property } from "lit/decorators";
+import type { SortableEvent } from "sortablejs";
+import { fireEvent } from "../common/dom/fire_event";
+import type { SortableInstance } from "../resources/sortable";
+
+declare global {
+  interface HASSDomEvents {
+    "item-moved": {
+      oldIndex: number;
+      newIndex: number;
+    };
+  }
+}
+
+@customElement("ha-sortable")
+export class HaSortable extends LitElement {
+  private _sortable?: SortableInstance;
+
+  @property({ type: Boolean }) public disabled = false;
+
+  @property({ type: String }) public container?: string;
+
+  @property({ type: String }) public item?: string;
+
+  @property({ type: String }) public handle?: string;
```

Comment:
I would make it cleared that this is a selector, like name it `itemSelector`, `containerSelector`, `handleSelector`, or make it accept both a selector string and an element

Reply:
I will rename it to selector 🙂

Reply:
container could be an element I guess, but selectors are probably easier

Reply:
It's not used and I don't think it will be used. Picking the first child is ok.